SpatialGDK/Source/SpatialGDKFunctionalTests/SpatialGDK/SpatialTestPossession/SpatialTestRemotePossession.cpp

```diff
@@ -61,3 +76,28 @@ void ASpatialTestRemotePossession::AddWaitStep(const FWorkerDefinition& Worker)
 		WaitTime += DeltaTime;
 	});
 }
+
+void ASpatialTestRemotePossession::AddCleanStep()
```

Comment:
FYI there is a mechanism for reusing test steps called `FSpatialFunctionalTestStepDefinition`, see SpatialAuthorityTest for a usage example. You could define the Clean step as an `FSpatialFunctionalTestStepDefinition `and then use `AddStepFromDefinition` wherever the step is needed.